### PR TITLE
Fix documentation warnings and improve docstring consistency.

### PR DESCRIPTION
**Pull request by MaxJPRey**

### CODE REVIEW

File: pyaedt/edb.py [hunk 1]
```diff
@@ -621,7 +621,8 @@ def core_stackup(self):
         """Core stackup.
 
         .. deprecated:: 0.6.5
-        There is no need to use core_stackup anymore. You can instantiate new class stackup directly from edb class .
+            There is no need to use core_stackup anymore.
+            You can instantiate new class stackup directly from edb class .
```
Comment (PipKat) [inline pyaedt/edb.py:625]:
```suggestion
            There is no need to use ``core_stackup`` anymore.
            You can instantiate a new ``stackup`` class directly from the ``Edb`` class.
```

File: pyaedt/edb_core/stackup.py [hunk 2]
```diff
@@ -502,7 +502,7 @@ class EdbStackup(object):
     """Manages EDB methods for stackup and material management accessible from `Edb.core_stackup` property.
 
     .. deprecated:: 0.6.5
-    This class has been deprecated and replaced by Stackup class.
+        This class has been deprecated and replaced by Stackup class.
```
Comment (PipKat) [inline pyaedt/edb_core/stackup.py:505]:
```suggestion
        This class has been deprecated and replaced by the ``Stackup`` class.
```

File: pyaedt/modules/PostProcessor.py [hunk 3]
```diff
@@ -3599,7 +3599,7 @@ def create_ami_statistical_eye_plot(
         setupname : str
             Name of the setup.
         probe_id : str
-            AMI Probe Name to use
+            AMI Probe Name to use.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3602]:
```suggestion
            AMI probe name to use.
```

File: pyaedt/modules/PostProcessor.py [hunk 4]
```diff
@@ -3495,18 +3495,18 @@ def create_ami_initial_response_plot(
...
-            AMI Probe name to use
+            AMI Probe name to use.
         variation_list_w_value : list
-            list of variations with relative values
-        plot_type : str, Default ``"Rectangular Plot"``
+            List of variations with relative values.
+        plot_type : str
             String containing the report type. Default is ``"Rectangular Plot"``. It can be ``"Data Table"``,
             ``"Rectangular Stacked Plot"``or any of the other valid AEDT Report types.
+            Default value is ``"Rectangular Plot"``.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3509]:
```suggestion
            The default is ``"Rectangular Plot"``.
```

File: pyaedt/modules/PostProcessor.py [hunk 5]
```diff
@@ -3495,18 +3495,18 @@ def create_ami_initial_response_plot(
...
 
-
         Parameters
         ----------
         setupname : str
-            Name of the setup
+            Name of the setup.
         ami_name : str
-            AMI Probe name to use
+            AMI Probe name to use.
```
Comment (PipKat) [inline pyaedt/modules/PostProcessor.py:3503]:
```suggestion
            AMI probe name to use.
```

File: pyaedt/modeler/Model3DLayout.py [hunk 6]
```diff
@@ -159,8 +159,8 @@ def primitives(self):
         """Primitives.
 
         .. deprecated:: 0.4.15
-        There is no need to use primitives anymore. You can instantiate methods for
-        primitives directly from the modeler.
+            There is no need to use primitives anymore. You can instantiate methods for
+            primitives directly from the modeler.
```
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:163]:
```suggestion
            There is no need to use the ``primitives`` property anymore. You can instantiate methods for
            primitives directly from the modeler.
```

File: pyaedt/modules/PostProcessor.py [hunk 7]
```diff
@@ -3820,7 +3820,7 @@ def sample_waveform(
             List with clock tics. The default is ``None``, in which case the clock tics from
             the AMI receiver are used.
         pandas_enabled : bool, optional
-            ``True`` if data format is pandas. The default is `False``
+            ``True`` if data format is pandas. The default is ``False``.
```
[the remark this answers is not in the document]
Comment (PipKat) [reply pyaedt/modules/PostProcessor.py:3823]: You might want to link "Pandas" to a topic on the Pandas website.
All descriptions for Booleans should start with a ``Whether to ...."

[the remark this answers is not in the document]
Comment (MaxJPRey) [reply pyaedt/modules/PostProcessor.py:3823]: OK @PipKat. I will try to modify the descriptions not starting with the "Wheter to ..." when they should.

**Review by maxcapodi78 — APPROVED** [review]

**Pull request merged**